scripts/generated_file_list.txt

```diff
@@ -0,0 +1,10 @@
+accounts/abi/bind/precompilebind/precompile_config_template.go
+accounts/abi/bind/precompilebind/precompile_config_test_template.go
+accounts/abi/bind/precompilebind/precompile_contract_template.go
+accounts/abi/bind/precompilebind/precompile_contract_test_template.go
+accounts/abi/bind/precompilebind/precompile_event_template.go
+accounts/abi/bind/precompilebind/precompile_module_template.go
```

Comment:
why did we special case this? I think these can be excluded in `scripts/upstream_files.txt` and then we can use avalanche header right?